.github/workflows/knative-downstream.yaml

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Downstream
+
+on:
+  push:
+    branches: [ 'master', 'release-*' ]
```

Comment:
should this first on branches or just PRs?

Reply:
I think PRs will be useful and I am not sure about branches. Let me remove the `push` config.